db/migrate/20180507103431_create_restaurants.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRestaurants < ActiveRecord::Migration[5.2]
+  def change
+    create_table :restaurants do |t|
+      t.string :name
+      t.string :cuisine
```

Comment:
I thought about it, I decided to go with 1 cuisine per restaurant at the moment because my focus is the client side. I might add this later, I want to get to the react side first and if I'll have time I'll get back to it.

Reply:
I also think that creating a cuisine entity is the right way to go, even more so if you say you want to put some emphasis on the front end stuff